amp/WEB-INF/src/org/dgfoundation/amp/onepager/web/pages/AmpHeaderFooter.java

```diff
@@ -60,6 +61,15 @@ public AmpHeaderFooter() {
         add(new DocumentReadyBehavior());
         AmpAjaxBehavior ampajax = new AmpAjaxBehavior();
         add(ampajax);
+        //timer to make a dummy request to keep page session alive.. for testing
+        add(new AbstractAjaxTimerBehavior(Duration.seconds(30)) {
```

Comment:
@jdeanquin-dg ..review this and advise .. This is a dummy request to keep the wicket session alive